<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>OpenCV MewTracking</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabCONTROLLER">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>331</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabCAMS">
     <attribute name="title">
      <string>CAMERAS</string>
     </attribute>
     <widget class="QGroupBox" name="gbCAMERAS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>CAMERAS</string>
      </property>
      <widget class="QListWidget" name="listCAMERAS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnCAMfind">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../My Documents/GraphGale/LUPA1.png</normaloff>../../../../My Documents/GraphGale/LUPA1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>40</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCAMadd">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add IPCAM</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCAMpreview">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../My Documents/GraphGale/preview1.png</normaloff>../../../../My Documents/GraphGale/preview1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCAMremove">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../My Documents/GraphGale/TRASH1.png</normaloff>../../../../My Documents/GraphGale/TRASH1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtIPCAMinput">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>80</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHSV">
     <attribute name="title">
      <string>COLOR FILTER</string>
     </attribute>
     <widget class="QPushButton" name="btnHSVpreview">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview OFF</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbHSVfilter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>HSV FILTER</string>
      </property>
      <widget class="QFrame" name="frameHSV">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>501</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>0</y>
          <width>131</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutSliderMin">
         <item>
          <widget class="QSlider" name="sliderHUEmin">
           <property name="maximum">
            <number>179</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderSATmin">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderVALmin">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>0</y>
          <width>131</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="LayoutSliderMax">
         <item>
          <widget class="QSlider" name="sliderHUEmax">
           <property name="maximum">
            <number>179</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>179</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderSATmax">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderVALmax">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>126</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutLBLmin">
         <item>
          <widget class="QLabel" name="lblHUEmin">
           <property name="text">
            <string>HUE &gt;&gt; MIN: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSATmin">
           <property name="text">
            <string>SAT &gt;&gt; MIN: 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVALmin">
           <property name="text">
            <string>VAL &gt;&gt; MIN: 0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>0</y>
          <width>80</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutLBLmax">
         <item>
          <widget class="QLabel" name="lblHUEmax">
           <property name="text">
            <string>MAX: 179</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSATmax">
           <property name="text">
            <string>MAX: 255</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVALmax">
           <property name="text">
            <string>MAX: 255</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbCOLORselec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>101</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>SELECT COLOR</string>
      </property>
      <widget class="QPushButton" name="btnCOLOURpicker">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Color Picker</string>
       </property>
      </widget>
      <widget class="QFrame" name="frameRGBselector">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbCOLORname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>COLOR NAME</string>
      </property>
      <widget class="QLineEdit" name="txtNAMEcolour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbCOLORlist">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>181</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>COLOR LIST</string>
      </property>
      <widget class="QPushButton" name="btnRmColour">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEditColour">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listCOLOUR">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnAddColour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTRACKING">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>COLOR TRACKING</string>
     </attribute>
     <widget class="QGroupBox" name="gbTRACKINGpreview">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>PREVIEW</string>
      </property>
      <widget class="QPushButton" name="btnTRACKING">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>TRACKING
WITH CAM</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCAMchroma">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>TRACKING CHROMA</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
